src/content/docs/uk/getting-started/installation.md

```
@@ -0,0 +1,124 @@
+```
+
+:::
+
+:::note
+Скористайтеся своїм списком із `Scripts/pkg_extra.lst`,
+або виконайте `cp  Scripts/pkg_extra.lst Scripts/pkg_user.lst`, якщо бажаєте встановити всі додаткові пакунки.
+:::
+
+## Детальне та ручне встановлення
+
+#### Клонування
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)